python/src/aac/plugins/lsp_server/providers/rename_provider.py

```diff
@@ -52,10 +52,9 @@ def get_rename_edits(
 
         if document:
             symbol = get_symbol_at_position(document.source, position.line, position.character)
+            edits = self._get_rename_edits(symbol, new_name) if symbol else {}
 
             if symbol:
```

Comment:
I'm pretty sure this if statement is not longer needed since `edits` will be set to a valid value depending on whether `symbol` is set. Nothing else uses symbol in the `try`/`except` block.

Reply:
The if state is necessary since the response to a null symbol is a None response rather than a WorkspaceEdit with 0 changes.